README.md

```diff
@@ -10,8 +10,9 @@ Library for application monitoring. It's abstraction of metrics inspired by [Dro
 
 Main advantages of this library:
 1) Universal abstraction with misc. implementations
-1) Support of multiple exports at once (`MultiMonitor`)
-1) Scala API
+2) Support of multiple exports at once (`MultiMonitor`)
+3) Scala API
+4) Cats-Effect API
```

Comment:
Just to be consistent - do you want to call it `Cats-Effect` or `Scala Effect` ?